app/src/main/java/com/hadiyarajesh/notex/di/DatabaseModule.kt

```diff
@@ -35,6 +37,11 @@ class DatabaseModule {
     @Singleton
     fun provideReminderDao(db: NoteXDatabase): ReminderDao = db.reminderDao
 
+    @Provides
+    @Singleton
+    fun provideMainScheduler(db: ReminderDao): MainNotificationWorker = ReminderWorkManager(db)
```

Comment:
Use Proper naming convention for any variable.
```suggestion
    fun provideMainScheduler(reminderDao: ReminderDao): MainNotificationWorker = ReminderWorkManager(reminderDao)
```